docs/themes/hugo-geekdoc/layouts/_default/terms.html

```diff
@@ -19,8 +19,8 @@ <h1 class="gdoc-post__title">
         <span class="flex align-center no-wrap">
           <svg class="gdoc-icon gdoc_star"><use xlink:href="#gdoc_star"></use></svg>
           <span>
-            {{ $latet := index .Pages.ByDate 0 }}
-            {{ with $latet }}
+            {{ $latest := index .Pages.ByDate 0 }}
+            {{ with $latest }}
```

Comment:
This is from the Hugo theme, which we should not change directly. Do you know if updating the theme will resolve the issue?